lang/syntax/src/trees/generic/lookup_table.rs

```diff
@@ -1,8 +1,6 @@
 use std::fmt;
 
-use data::HashMap;
-use data::HashSet;
-
+pub type HashSet<V> = fxhash::FxHashSet<V>;
```

Comment:
Couldn't `HashSet`  be imported from the `syntax::common` module here?

Reply:
Done